sprint/.ddev/docker-compose.sprint.yml

```diff
@@ -0,0 +1,43 @@
+version: "3"
```

Comment:
Can you explain why this is added as a ddev service? It doesn't seem to me it has anything to do with ddev, and putting it in .ddev directory can cause a lot of confusion. If it's not a ddev service of some type can we put it in .quicksprint or something like that?

Reply:
Can it run through ddev-router without it being a ddev service? I think we were both independently following the docs about how to add containers so that we could start up the sprint toolset in one docker-compose command.

Reply:
Is there a more suitable way to do this?

Reply:
As @mradcliffe said I was following ddev docs here..

Reply:
@rfay is there a more suitable way that allows them to get started at the same time as upping the drupal 8 site via ddev?

Reply:
The docs about adding services are for adding things like solr. Here we're adding a user-oriented web-based text editor right? Maybe I just don't understand the purpose, but I thought this was just a web-based editor, and is not integrated with ddev.

I may be lost, but if it's what I understand I'd bring up bring up the editor container with a completely separate and decoupled docker-compose command, and then bring up ddev.  I don't understand what they have to do with each other.

Reply:
@rfay can we make use of the ddev-router if it's not so closely integrated?

I specifically don't want it to have a different URL, as the user may have multiple instances running at once and it should be tied to the correct instance.

I just followed the ddev docs.. thats why it ended up this way..